odoo/addons/base/wizard/base_language_install_views.xml

```diff
@@ -2,39 +2,53 @@
 <odoo>
     <data>
 
+        <record id="base.language_install_view_form_lang_switch" model="ir.ui.view">
```

Comment:
ID should contain only dots (base.language.install.view.form.lang.switch) as per mighty guidelines !